lib/core/feature-flags/src/lib/directives/features.directive.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Directive, Inject, Input, OnDestroy, TemplateRef, ViewContainerRef } from '@angular/core';
+import { BehaviorSubject, Subject, combineLatest } from 'rxjs';
+import { IFeaturesService, FeaturesServiceToken, FlagChangeset } from '../interfaces/features.interface';
+import { takeUntil } from 'rxjs/operators';
+
+@Directive({
+    /* eslint-disable-next-line @angular-eslint/directive-selector */
+    selector: '[forFeatures]',
```

Comment:
Should I rename this directive and the one below to 
`adf-for-feature` and `adf-not-for-features`?
@DenysVuika

Reply:
I'm not sure about this, what do you think @DenysVuika . Sometimes the ADF prefix for directives is more bothering then helping, I think this is a similar case.
From my side, I would leave it as they are, since this way they are quite easy to read and self-explanatory.

Reply:
can be `adfForFeatures`, similar to how we have all `cdkFeature` directives from material